### ISSUE TITLE
Camera capture request update causes app to crash

### ISSUE THREAD

**Issue by tokonuulu** [twilio/video-quickstart-android#608]
# Description
Updating capture request after camera switch causes app to crash with the following log. I am using [CameraCaptureCompat](https://github.com/twilio/twilio-video-app-android/blob/master/app/src/main/java/com/twilio/video/app/util/CameraCapturerCompat.kt).

# Steps to reproduce

1. Start capturing
3. Update capture request (works well)
3. Switch camera
4. Update capture request (crash)

Crashes 100% of the time.
VideoSDK version: 6.0.1

```
E/AndroidRuntime: FATAL EXCEPTION: CaptureThread
    java.lang.NullPointerException: Attempt to invoke virtual method 'android.hardware.camera2.CaptureRequest$Builder android.hardware.camera2.CameraDevice.createCaptureRequest(int)' on a null object reference
        at tvi.webrtc.Camera2Session.configureCaptureRequestBuilder(Camera2Session.java:369)
        at com.twilio.video.Camera2Capturer.updateCaptureRequestOnCameraThread(Camera2Capturer.java:433)
        at com.twilio.video.Camera2Capturer.lambda$updateCaptureRequest$1$Camera2Capturer(Camera2Capturer.java:390)
        at com.twilio.video.-$$Lambda$Camera2Capturer$CcvEP_UV-5a3qu8wTCCcc6bukzY.run(Unknown Source:4)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:237)
        at android.os.HandlerThread.run(HandlerThread.java:67)
```

**Comment by khoamt** [comment]
I also face the same issue.

**Comment by Alton09** [comment]
@khoamt @tokonuulu This issue has been fixed with the latest release, 6.2.1. Closing this issue for now, but please feel free to open a new issue as needed. Thanks!
